File: Day10/TwoDArray.py
arr=[]
row=3
clmn=3

# Initialize the array elements 
print("Enter the array elements:")
for i in range (row):
    temp=[]
    for j in range (clmn):
        ele=int(input(f"arr[{i}][{j}]="))
        temp.append(ele)
    arr.append(temp)

# Display the array elements
print("Display the array elements:")
for i in range (0,len(arr)):
    for j in range (0,len(arr[0])):
        print(arr[i][j],end=" ")
    print("")

# Search an element in 2 D array
key=int(input("Enter the element to search:"))
flag=0
for i in range (0,len(arr)):
    for j in range (0,len(arr[0])):
        if(arr[i][j]==key):
            flag=1
            print("Element is present")
            break
if(flag==0):
    print("Element is not present")

File: Day10/TwoDArray.java
package Day10;
import java.util.Scanner;
public class TwoDArray {
   public static void main(String[] args) {
     Scanner sc=new Scanner(System.in);
     System.out.println("Enter the no of row:");
     int row=sc.nextInt();
     System.out.println("Enter the no of clmn:");
     int clmn=sc.nextInt();
     int arr[][]=new int [row][clmn];

     //Intialize the array elements
     System.out.println("Enter the array elements:");
     for(int i=0;i<row;i++)
     {
        for(int j=0;j<clmn;j++)
        {
            System.out.print("arr["+i+"]["+j+"]=");
            arr[i][j]=sc.nextInt();
        }
     }

     //Display the array elements
     System.out.println("Array elements are:");
      for(int i=0;i<arr.length;i++)
     {
        for(int j=0;j<arr[0].length;j++)
        {
            System.out.print(arr[i][j]+" ");  
        }
        System.out.println("");
     }

     //Search an element inside the 2 D array
     System.out.println("enter the value to search");
     int key=sc.nextInt();
     int flag=0;
     for(int i=0;i<arr.length;i++)
     {
        for(int j=0;j<arr[0].length;j++)
        {
            if(arr[i][j]==key)
            {
                flag=1;
                System.out.println("Element is present");
                break;
            }    
        }
     }
     if(flag==0)
        System.out.println("Element is not present");

   }
}
